MapboxDirectionsTests/IntructionsTests.swift

```diff
@@ -68,15 +66,22 @@ class SpokenInstructionsTests: XCTestCase {
         let spokenInstructions = step.instructionsSpokenAlongStep!
         
         XCTAssertEqual(spokenInstructions[0].distanceAlongStep, 793.8)
-        XCTAssertEqual(spokenInstructions[1].distanceAlongStep, 304.0)
-        XCTAssertEqual(spokenInstructions[2].distanceAlongStep, 65.1)
+        XCTAssertEqual(spokenInstructions[1].distanceAlongStep, 348.2)
+        XCTAssertEqual(spokenInstructions[2].distanceAlongStep, 74.6)
         
-        XCTAssertEqual(spokenInstructions[0].ssmlText, "<speak>Head south on 8th Avenue</speak>")
-        XCTAssertEqual(spokenInstructions[1].ssmlText, "<speak>In 1000 feet, turn left onto John F Kennedy Drive</speak>")
-        XCTAssertEqual(spokenInstructions[2].ssmlText, "<speak>Turn left onto John F Kennedy Drive</speak>")
+        XCTAssertEqual(spokenInstructions[0].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">Head south on <say-as interpret-as=\"address\">8th</say-as> Avenue for a half mile</prosody></amazon:effect></speak>")
+        XCTAssertEqual(spokenInstructions[1].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">In a quarter mile, turn left onto John F Kennedy Drive</prosody></amazon:effect></speak>")
+        XCTAssertEqual(spokenInstructions[2].ssmlText, "<speak><amazon:effect name=\"drc\"><prosody rate=\"1.08\">Turn left onto John F Kennedy Drive</prosody></amazon:effect></speak>")
         
-        XCTAssertEqual(spokenInstructions[0].text, "Head south on 8th Avenue")
-        XCTAssertEqual(spokenInstructions[1].text, "In 1000 feet, turn left onto John F Kennedy Drive")
+        XCTAssertEqual(spokenInstructions[0].text, "Head south on 8th Avenue for a half mile")
+        XCTAssertEqual(spokenInstructions[1].text, "In a quarter mile, turn left onto John F Kennedy Drive")
         XCTAssertEqual(spokenInstructions[2].text, "Turn left onto John F Kennedy Drive")
+        
+        let visualInstructions = step.instructionsDisplayedAlongStep!
```

Comment:
If `instructionsDisplayedAlongStep` happens to be `nil`, the test will crash and you’ll get a pretty unintelligible error on CI. At least add a non-nil assertion before force-unwrapping, and consider chaining optionals to avoid having to force-unwrap.